src/main/sensors/battery.c

```diff
@@ -434,6 +434,14 @@ uint8_t getBatteryCellCount(void)
     return batteryCellCount;
 }
 
+/**
+ * Gets average battery cell voltage in 0.01V units.
+ */
+uint16_t getBatteryAverageCellVoltage(void)
```

Comment:
Yeah, I'd probably go for that. We don't know what precision is required if cell voltage is used in other places.